admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -1503,3 +1503,51 @@ which are older then the number of days that is set for ``activity_expire_days``
 	'wnd.logging.enable' => true,
 
 This enables debug logs for the windows_network_drive app.
+
+Overriding Existing Parameter Values Using Environment Variables
+----------------------------------------------------------------
+
+ownCloud supports overriding existing configuration values for the web UI,
+command line, and Cron environments, using `environment variables`_.
+Doing so avoids the need to storing credentials, and other *private* data,  data
+which you never want to leak out, in either code or source repositories. 
+What’s more, by taking this approach, code changes aren’t required to deploy to
```

Comment:
This is the only sentence that i would like to change, as it just sounds a bit confusing. We talked about that earlier, and you told me that the intent of this sentence is something like:

Whenever you deploy to multiple environments, you do not have to have different deployments for different server configurations (e.g. database connections), because those are already configured via environment variables.

So something along those lines should be way more straight forward.